enzy_htp/preparation/align_ligand.py

```diff
@@ -0,0 +1,196 @@
+"""Functionality for aligning the coordinates of a ligand to a target structure using mapped PDB-style atom names. Note that atoms 
+without names will not be easily aligned. Only align_ligand() should be called. All other functions are for implementation purposes 
+only and SHOULD NOT be called directly by the user.
+
+Author: Chris Jurich <chris.jurich@vanderbilt.edu>
+Date: 2023-10-09
+"""
+
+from collections import defaultdict
+from typing import List, Dict, Set, Tuple
+from pathlib import Path
+
+import pandas as pd
+import numpy as np
+import rdkit.Chem as _rchem
+from rdkit.Chem import rdMolTransforms as rdmt
```

Comment:
Can the use of these go to RDKitInterface as well as the data structure conversion (Structure() -> Molecule())?

Reply:
I agree. All access to `rdkit` should be done through the `RDKItInterface()`